servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
Either way can work, one algorithm could be to scan through all the blocks by LRU and bucket the blocks by `StorageDir` (ie. with a hashmap) until one `StorageDir` has enough space, then you can evict all the blocks in that bucket.